### ISSUE TITLE
Resources not loaded on HTTPS

### ISSUE THREAD

**Issue by mksplg** [rstacruz/flatdoc#27]
When flatdoc is served using HTTPS, [Firefox 23 and up](https://developer.mozilla.org/en-US/docs/Security/MixedContent/How_to_fix_website_with_mixed_content) and Chrome block the content with the message `Blocked loading mixed active content "..."`.

Updating the links in the template solves most of the issues except the [included font](https://github.com/rstacruz/flatdoc/blob/gh-pages/theme-white/style.css#L4) in theme-white's style.css.

**Comment by rstacruz** [comment]
Nice, anyone got a PR for this?

**Comment by mksplg** [comment]
The resource links would need to be changed to github's raw URLs, e.g. https://raw.githubusercontent.com/rstacruz/flatdoc/gh-pages/v/0.8.1/flatdoc.js as [links currently used](http://rstacruz.github.io/flatdoc/v/0.8.1/flatdoc.js) in the template are redirected to http://ricostacruz.com which does not support https.

Unfortunately, ghbtns.com [used in the template](https://github.com/rstacruz/flatdoc/blob/gh-pages/templates/template.html#L55) also does not support https yet, see https://github.com/mdo/github-buttons/issues/61.

**Comment by rstacruz** [comment]
Oh, right. Forgot about that — gh-pages doesn't support HTTPS.

However, IIRC GitHub was pretty clear on not using `https://raw.githubusercontent.com` on production sites. Perhaps there isn't a good way around this afterall, unless you host it yourself.

**Comment by mksplg** [comment]
You're right, `https://raw.githubusercontent.com` serves everything with content-type `text/plain`.

Wouldn't https on gh-pages work if http://ricostacruz.com supported it or is that not supported by the redirect? At least if the page is accessed with https://user.github.io/repository without redirects, https works fine.

**Comment by rstacruz** [comment]
Google Fonts is now loaded via HTTPS. c1dbf23

**Comment by rstacruz** [comment]
rawgit.com offers a free CDN for git repos, and it support HTTPS. You should be able to use this URL:

https://cdn.rawgit.com/rstacruz/flatdoc/v0.8.6/flatdoc.js
